tests/test_utils.py

```
@@ -126,6 +125,15 @@ def init_page_and_wait_ready(
         pass
 
     page.goto(url, wait_until="domcontentloaded", timeout=DEFAULT_TIMEOUT)
+
+    # Dismiss GPU warning modal if displayed in software rasterizer/headless CI
+    gpu_btn = page.locator("#gpu-warning-dismiss-btn")
+    try:
+        gpu_btn.wait_for(state="visible", timeout=1500)
+        gpu_btn.click()
+    except Exception:
+        pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)